source/facter/3.1/release_notes.md

```diff
@@ -282,3 +282,14 @@ The `puppet-agent` package that installs Facter 3.0.0 did not create the `rubysi
 
 * [FACT-1154](https://tickets.puppetlabs.com/browse/FACT-1154)
 
+### BREAK: Removed Ruby Facter implementations
```

Comment:
I think this shouldn't be in the Facter 3.1 release notes, since the behavior was changed in 3.0. I'd be :+1: on putting a note in the Facter 3.1 custom facts guides, but :-1: to putting it in the release notes.